metadata-db-app/src/cmr/metadata_db/data/oracle/generic_documents.clj

```diff
@@ -90,16 +91,24 @@
   (apply dissoc document fields-vec))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
+;; TODO: Generic work: shouldn't we use the same number generator as cmr.metadata-db.data.oracle.concepts/ generate-concept-id
+;; seq-num (:nextval (first (su/query db ["SELECT concept_id_seq.NEXTVAL FROM DUAL"])))
 (defn generate-concept-id
   [db document]
   (let [{:keys [concept-type provider-id]} document
+        concept-sub-type (get document "concept-sub-type")
         raw-count (-> db
                       (jdbc/query ["SELECT count(DISTINCT concept_id) AS last FROM CMR_GENERIC_DOCUMENTS"])
                       first
                       :last)
         next (+ 1200000001 raw-count)]
```

Comment:
Would prefer if this magic number was moved to a def with a docstring explaining why it's the value it is.

Reply:
We will have to ask Thomas - what I really want to do is to the same as 
cmr.metadata-db.data.oracle.concepts/ generate-concept-id
seq-num (:nextval (first (su/query db ["SELECT concept_id_seq.NEXTVAL FROM DUAL"])))

I think this would be better.

Reply:
@jceaser